pyquil/api/_compiler.py

```diff
@@ -356,6 +356,23 @@ def reset(self) -> None:
         """
         self._qpu_compiler_client = None
 
+    @property
+    def client(self) -> Client:
+        """Return the `Client` for the compiler (i.e. quilc, not translation service)."""
+        # TODO(notmgsk): This was introduced around 2.25 to provide
+        # feature parity with QVMCompiler which provides a client
+        # property, whereas QPUCompiler provides a quilc_client
+        # property.
+        return self.quilc_client
+
+    def set_timeout(self, timeout: float) -> None:
```

Comment:
I always appreciate units in the signature for timeouts (even though you already have it in compiler.rst). So either `timeout: datetime.duration`, `timeout_seconds: int`, or more likely (for consistency) an explicit comment in the docstring.

Reply:
I'll go with the docstr :)